_til/linux/install-ubuntu.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Install Ubuntu 
+category: linux
+---
+
+Do you want to move from Windows into a UNIX like dev environment? Ubuntu is an easy way to get your feet wet and try out Linux.
+
+## Basic Installation
+
+Ubuntu ships with a very straight forward installer and makes set up a snap. Here are the only steps:
+* Choose a language and keyboard layout (Usually English-US for both)
+* Connect to a network (Optional, convenient to install updates during installation)
+* Choose to install updates
+* Choose to install 3rd party non-free items (Flash, MP3, etc. I usually select yes as it makes the user experience much easier)
```

Comment:
make a note here that this is free as in libre and not something you need to pay for